src/Instance/ItemsList/ItemsList.js

```diff
@@ -60,7 +60,7 @@ const getFormatter = (
         <ItemBarcode
           item={item}
           holdingId={holding.id}
-          instanceId={holding.instanceId}
+          instanceId={holding.instance}
```

Comment:
This looks wrong to me. `instanceId` certainly needs to be a string, and I would expect `holding.instance` to be an object, though I haven't been able to trace where that property is assigned (it's not part of the response from `holdings-storage/holdings/${someId}`.

Reply:
The `holding` object that's passed into `<Barcode />` doesn't have a property of `instanceId`. And I can't see it being modified from when it's [fetched](https://github.com/frontside-folio/ui-inventory/blob/996a2cdf08d45903c215609f441d7fc89f2ee8a8/src/Instance/HoldingsList/HoldingsListContainer.js#L21) from `<HoldingsListContainer />` all the way to where it's received in `<Barcode />`.

![Screen Shot 2020-07-17 at 4 18 09 PM](https://user-images.githubusercontent.com/29791650/87827586-2c529980-c849-11ea-99ab-03167db29223.png)

Reply:
@minkimcello the `instanceId` field exists on the holding record. Please check the responses from https://folio-snapshot.aws.indexdata.com

![holding](https://user-images.githubusercontent.com/63545/87943791-8a12fb80-ca6c-11ea-9b87-d360248e4ccb.png)